source/integrations/zapier.rst

```diff
@@ -0,0 +1,147 @@
+Zapier and Mattermost
+===================
+
+Integrate Mattermost with over 500 supported apps on Zapier, including `GitHub <https://zapier.com/zapbook/github/>`_, `Jira <https://zapier.com/zapbook/jira/>`_, `Wufoo <https://zapier.com/zapbook/wufoo/>`_, `Salesforce <https://zapier.com/zapbook/salesforce/>`_, `Twitter <https://zapier.com/zapbook/twitter/>`_, `Gmail <https://zapier.com/zapbook/gmail/>`_ and `many more <https://zapier.com/zapbook/>`_. Zapier integration triggers new Mattermost messages for events in other apps. This documentation includes:
+
+- `Zapier Setup Guide <https://docs.mattermost.com/integrations/zapier.html#zapier-setup-guide>`_ - **Instructions on registering the Zapier app on your server and creating a zap**
+- `Message Formatting Tips <https://docs.mattermost.com/integrations/zapier.html#message-formatting-tips>`_ - **Tips from the Mattermost team on formatting Zapier integration messages**
+- `Troubleshooting Guide <https://docs.mattermost.com/integrations/zapier.html#troubleshooting-guide>`_ - **Advice on troubleshooting common setup issues**
+
+
+Zapier Setup Guide
+--------------------------------
+Zapier is authorized using OAuth2.0. The setup guide requires that a System Admin register the Zapier app on their Mattermost server and can then optionally allow any users on the system to create Zapier integrations.
```

Comment:
I think Lindy is right, we should mention you need to sign up for a Zapier account at the beginning here (it's really only any user with a zapier account who can set it up, not any user)